runtime/autoload/dist/ft.vim

```diff
@@ -112,6 +112,25 @@ func dist#ft#BindzoneCheck(default)
   endif
 endfunc
 
+" Returns true if file content looks like RAPID
+func IsRapid(sChkExt = "")
```

Comment:
Is that valid in legacy vimscript?

Reply:
Yes. There is an example in `:h optional-function-argument`